packages/ods/src/components/tooltip/src/components/ods-tooltip/ods-tooltip.tsx

```diff
@@ -54,19 +60,47 @@ export class OdsTooltip {
     this.triggerElement?.addEventListener('blur', this.boundHide);
     this.triggerElement?.addEventListener('focus', this.boundShow);
     this.triggerElement?.addEventListener('mouseenter', this.boundShow);
-    this.triggerElement?.addEventListener('mouseleave', this.boundHide);
+    this.triggerElement?.addEventListener('mouseleave', this.boundLeaveTrigger);
+    this.el.addEventListener('mouseenter', this.boundShowOnTooltip);
+    this.el.addEventListener('mouseleave', this.boundHideOnTooltip);
   }
 
   disconnectedCallback() : void {
     this.triggerElement?.removeEventListener('blur', this.boundHide);
     this.triggerElement?.removeEventListener('focus', this.boundShow);
     this.triggerElement?.removeEventListener('mouseenter', this.boundShow);
-    this.triggerElement?.removeEventListener('mouseleave', this.boundHide);
+    this.triggerElement?.removeEventListener('mouseleave', this.boundLeaveTrigger);
+    this.el.removeEventListener('mouseenter', this.boundShowOnTooltip);
+    this.el.removeEventListener('mouseleave', this.boundHideOnTooltip);
+  }
+
+  private leaveTrigger(): void {
+    setTimeout(() => this.hideByTooltipHover(), 50);
+  }
+
+  private showOnElEnter(): Promise<void> {
+    this.isTooltipHover = true;
+    return this.boundShow();
+  }
+
+  private hideOnElLeave(): void {
+    this.isTooltipHover = false;
+    this.timer = setTimeout(() => this.hideByTooltipHover(), 50);
```

Comment:
you could add the `setTimeout` in the `hideByTooltipHover` directly, and then remove the `leaveTrigger` method